index.js

```diff
@@ -202,36 +211,36 @@ function getOrientationChangeEvent (name, props) {
   return orientationChangeEvt;
 }
 
-function createOwnDelegate () {
+function createOwnDelegate(orientation) {
   var ownDelegate = Object.create({
-    addEventListener: function addEventListener (evt, cb) {
+    addEventListener: function addEventListener(evt, cb) {
       if (!this.listeners[evt]) {
         this.listeners[evt] = [];
       }
       if (this.listeners[evt].indexOf(cb) === -1) {
         this.listeners[evt].push(cb);
       }
     },
-    dispatchEvent: function dispatchEvent (evt) {
```

Comment:
I feel like I should introduce some code linting for this project

Reply:
Code linting would be of great assistance! I don't mind opening the PR to introduce code linting if you just want to point me towards what set of rules you'd like to configure :)

Reply:
Cool, that would be fantastic! Sorry for the long delay, this has been a busy time.
I think I’m good with just the prettier/eslint default rules except for using single quotes :)